### DISCUSSION TITLE
DurationCmd.readReply does not respect the precision

### DISCUSSION THREAD

**Discussion by bubunyo** [redis/go-redis#1818] [Ideas]
[`DurationCmd.readReply`](https://github.com/adjust/backend/blob/c0e728e0f4efb236e40740f6fc6773ead0346b95/vendor/github.com/go-redis/redis/v8/command.go#L559) does not respect the precision argument passed the DurationCmd for keys without an associated expire or do not exist. Is there any particular reason for this? I think it should.

**Comment by monkey92t** [comment]
Expired or non-existent keys have no meaning to `DurationCmd`, such as `TTL` commands
